Task3Math2/Task3Math2/Program.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task3Math2
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Условие Задачи 21: Дано четырехзначное число (ввод с клавиатуры). Найти число, полученное из исходного путем вычеркивания третьей цифры и приписывания ее в начало числа.\r\n");
+            Console.WriteLine("Введите четырёхзначное число: \r\n");
+
+            int Main_number = int.Parse(Console.ReadLine());
```

Comment:
В C# локальные переменные принято называть с маленькой буквы. Кроме того принят принцип именования camelCase, а не snake_case
```suggestion
            int mainNumber = int.Parse(Console.ReadLine());
```